cmd/cli/kubectl-kyverno/commands/apply/command.go

```diff
@@ -639,7 +644,11 @@ func (c *ApplyCommandConfig) loadPolicies() (
 			var gitPathToYamls string
 			c.GitBranch, gitPathToYamls = common.GetGitBranchOrPolicyPaths(c.GitBranch, repoURL, path)
 			fs := memfs.New()
-			if _, err := gitutils.Clone(repoURL, fs, c.GitBranch); err != nil {
+			auth := &http.BasicAuth{
```

Comment:
why is it defined as pointer when its not used as a pointer?

Reply:
Good point, fixed that.